packages/wrangler/bin/wrangler.js

```diff
@@ -2,15 +2,15 @@
 const { spawn } = require("child_process");
 const path = require("path");
 
-const MIN_NODE_VERSION = "18.0.0";
-
+const ERR_NODE_VERSION = "18.0.0";
+const MIN_NODE_VERSION = "20.0.0";
 let wranglerProcess;
 
 /**
  * Executes ../wrangler-dist/cli.js
  */
 function runWrangler() {
-	if (semiver(process.versions.node, MIN_NODE_VERSION) < 0) {
+	if (semiver(process.versions.node, ERR_NODE_VERSION) < 0) {
```

Comment:
This change doesn't seem correct to me, is it?

Why changing this to `ERR_NODE_VERSION`?

Reply:
I'm changing this to only _error_ if below 18, but tell people to upgrade to 20 in the messaging

Reply:
I'm not sure I'm following this... so if I were to use node 19 (which we don't support), why wouldn't we want to error here?

(also, is `< 0` correct? because this would not show the error if I were to use node precisely `18.0.0`)

Reply:
This is a tempory measure for the next few months to avoid being too loud about version support changes. We only stopped supporting Node.js 18 a couple of days ago, and this is the first change in Node.js version support we've made with the new v4 support guarantee.